### ISSUE TITLE
Maybe pass project to new maintainer?

### ISSUE THREAD

**Issue by cronventis** [pantsel/konga#673]
Hey Community...

So there has not been any update of Konga for quite a long time!
And I did a short research and noticed that the community is still using konga quite a lot and there are also a couple of PR's fixing stuff.

Since @pantsel does not seem to be working in this area anymore or does not seem to care (wich is totally fine) he might be willing to give this project to some one else?

I know this involves some research and time to find a responsible person to do so (and just to be clear, I'm just a user :-D), but i think this is a great project and just dont want it to stall.

Best regards.

**Comment by TomyCesaille** [comment]
Similar subjects: 

* https://github.com/pantsel/konga/issues/621
* https://github.com/pantsel/konga/issues/641

**Comment by cronventis** [comment]
Yep, I have seen that.. But if there is no reply.... What shall we do?

**Comment by TomyCesaille** [comment]
Anyone can fork and continue the adventure, the community will follow the fork if done correctly.
Nobody seems to have a maintained fork yet according to <https://techgaun.github.io/active-forks/index.html#https://github.com/pantsel/konga>

If the maintainer is still here a bit, he could pass the ownership so the project remains here.
But who takes over ? Real question.

**Comment by cronventis** [comment]
Yea.. But that "feels" wrong.. If someone just takes the project...
If the @pantsel just says: "contact me if you want to be the new maintainer", then everything goes an ordered way..
I would do it, if I had the skills.. but I'm working in a different area.. 
And yes.. The real question is who would....
